packages/core/src/index.html

```diff
@@ -248,6 +248,7 @@
             ></footer>
         </div>
         <div id="wallets_modal_root" class="modal-root"></div>
+        <div id="v2_modal_root" class="modal-root"></div>
```

Comment:
Just a question, soo all V2 packages can use this modal root?

Reply:
yeap